src/utils/grades.ts

```diff
@@ -0,0 +1,164 @@
+/** Ranked from easiest to hardest */
+
+import { useCallback, useMemo } from "react";
+import { useMeta } from "../components/common/meta";
+
+export const SORTED_GRADES = {
```

Comment:
@jossi87 - I didn't know whether the grades that come out of `/meta` are sorted or not. If they're sorted into a consistent order (easiest -> hardest or hardest -> easiest) then I can remove all of this boilerplate and generate this mapping at runtime.

Is the client able to rely on this being a stable sort?

Reply:
The grades from /meta is always ordered hardest -> easiest (consistent on every type: rock climbing, bouldering and ice climbing). You don't need the SORTED_GRADES :)

Reply:
Perfect - that'll make this much cleaner. I'll tidy this up :)